src/site/content/en/angular/creating-pwa-with-angular-cli/index.md

```diff
@@ -44,6 +44,17 @@ This command will:
 * Add the [`theme-color`](/themed-omnibox) `<meta>` tag to `index.html`.
 * Create app icons in the `src/assets` directory.
 
+## Register the service worker at startup
+
+In `app.module.ts`, make sure the service worker starts immediately so it can serve pages when offline:
```

Comment:
I am not super-familiar with `ngsw`, but I think this option just controls whether a first-time visitor will get the service worker **registered** immediately.

Once a service worker is registered, for repeat visitors, it will always start immediately prior to any network request made by an in-scope client.

Reply:
Changed, PTAL.